src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -58,9 +70,21 @@ public class WalletRecoveryRequest : RequestModel
         [Required(ErrorMessage = "A mnemonic is required.")]
         public string Mnemonic { get; set; }
 
+        /// <summary>
+        /// Supply the password that was used to create the wallet.
+        /// </summary>
+        /// <remarks>
+        /// If the account was created previously before passphrase was available, then the password was used as passphrase. It is therefor essential that
+        /// the <see cref="Passphrase"/> is not supplied for older accounts. Make sure <see cref="Passphrase"/> is set to null, not even empty string.
+        /// </remarks>
         [Required(ErrorMessage = "A password is required.")]
         public string Password { get; set; }
 
+        /// <summary>
+        /// Supply the passphrase that was used when account was created. If no passphrase was used, make sure this property is null, as empty string is a valid passphrase.
+        /// </summary>
+        public string Passphrase { get; set; }
```

Comment:
Can we create a `Passphrase` Type to allow us to enforce the explicit absence of a value?

Reply:
Could this type also include `Password` ? It seems they both work hand in hand.

Reply:
As @bokobza explained, that is how it is implemented. Implementing a Passphrase type could be an option, but that would mean that the existing Password property must/should be marked deprecated and eventually removed (or the API should be versioned).